src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -621,9 +621,10 @@ private function buildTableRows(array $rows): TableRows
                 if (!strstr($cell ?? '', "\n")) {
                     continue;
                 }
-                $escaped = implode("\n", array_map([OutputFormatter::class, 'escapeTrailingBackslash'], explode("\n", $cell)));
+                $eol = strstr($cell ?? '', \PHP_EOL) ? \PHP_EOL : "\n";
```

Comment:
This way:
- If the content of the cell is using EOL of the OS (ie `\r\n` for Windows os), we're using it.
- If the content of the cell is using `\n` we're not changing anything (this was the reason of the previous error https://ci.appveyor.com/project/fabpot/symfony/builds/49223425)